ledger/simulation/simulator.go

```diff
@@ -197,12 +197,15 @@ func (s Simulator) evaluate(hdr bookkeeping.BlockHeader, stxns []transactions.Si
 	}
 
 	// Finally, process any pending end-of-block state changes.
-	vb, err := eval.GenerateBlock()
+	ub, err := eval.GenerateBlock(nil) // XXX not fetching proposer addresses
```

Comment:
remove XXX?

Reply:
I have removed them and hopefully added worthwhile commentary explaining whty.